flink-core/src/main/java/org/apache/flink/api/common/state/CheckpointListener.java

```diff
@@ -0,0 +1,51 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Nit: The header here uses a different style than the rest of the code (other files have * at every line start).